### PR TITLE
NO MERGE: rpc helper no generated

### PR DESCRIPTION
**Pull request by chrisnojima**
reference for https://github.com/keybase/client/pull/3446

### CODE REVIEW

File: protocol/bin/flow.js [hunk 1]
```diff
@@ -198,7 +200,18 @@ function parseRecord (t) {
...
+      .filter(t => t)
+      .reduce((acc, t) => {
+        const clean = t[0].trim()
+        return acc.indexOf(clean) === -1 ? acc.concat([clean]) : acc
+      }, [])
+      .filter(t => t)
+      .sort()
+      .join('\n  | ')
+
+  console.log(rpcTypes)
```
Comment (chrisnojima) [inline protocol/bin/flow.js]: i'll remove this

File: protocol/bin/flow.js [hunk 2]
```diff
@@ -198,7 +200,18 @@ function parseRecord (t) {
...
 
 function makeRpcUnionType (typeDefs) {
-  const rpcTypes = typeDefs.map(t => t.match(/(\w*Rpc)/g)).filter(t => t).reduce((acc, t) => acc.concat(t), []).join('\n  | ')
+  const rpcTypes = typeDefs.map(t => t.match(/(\w*Rpc)/g))
+      .filter(t => t)
+      .reduce((acc, t) => {
+        const clean = t[0].trim()
+        return acc.indexOf(clean) === -1 ? acc.concat([clean]) : acc
+      }, [])
+      .filter(t => t)
```
Comment (MarcoPolo) [inline protocol/bin/flow.js]: We need this filter twice?

**Comment by MarcoPolo** [comment]
👍

File: protocol/bin/flow.js (hunk 2 above)
Comment (chrisnojima) [inline protocol/bin/flow.js]: removed: https://github.com/keybase/client/pull/3446/commits/10ab0ecd0f2a57bdddde30fdadacdd65d8beecf4

**Pull request closed without merging**